### PR TITLE
Add a README to the staff app

### PR DESCRIPTION
**Pull request by jchristgit**
Plus minor refactorings

### CODE REVIEW

File: pydis_site/apps/staff/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# The "staff" app
+
+This Django application hosts any staff-internal tooling, which, at time of
+writing, only is an endpoint to view logs uploaded by the Python bot.
```
Comment (wookie184) [inline pydis_site/apps/staff/README.md]:
```suggestion
writing, is only an endpoint to view logs uploaded by the Python bot.
```

**Review by wookie184 — COMMENTED** [review]
One comment, this is nice.

**Comment by jchristgit** [comment]
Thanks, applied it!

**Review by wookie184 — APPROVED** [review]

**Review by Xithrius — APPROVED** [review]
Thanks

**Pull request merged**